website/www/site/content/en/contribute/release-guide.md

```diff
@@ -46,10 +41,10 @@ The release process consists of several steps:
 1. Investigate performance regressions
 1. Create a release branch
 1. Verify release branch
+1. Triage release-blocking issues in JIRA
 1. Build a release candidate
-1. Vote on the release candidate
-1. During vote process, run validation tests
```

Comment:
It might be better to keep these steps explicityl here:
1. During vote process, run validation tests
1. If necessary, fix any issues and go back to step 3.

Reply:
+1